libs/application/templates/parental-leave/src/types.ts

```diff
@@ -20,3 +20,20 @@ export interface Payment {
     paid: boolean
   }
 }
+
```

Comment:
I believe we shouldn't have to create these types and we can use the types generated out of the vmst-client through openapi

Reply:
I moved these over from `ParentalLeaveForm.ts` so that I could also use them in the `Review` screen as well. Do you know how we can try to see if they are generated? Was just sitting there before.

Reply:
I would say it has been created before Bæring creates the VMST client so we probably added it manually at first. Unfortunately, I don't see any automatic way to find that. I guess the easier would be to run the swagger from vmst and see what it returns and check if it's what you need. It's very manual steps. Maybe @baering knows better way

Reply:
Sounds good, I added TODO comment for us to investigate for the next PR.